keyboards/0xcb/1337/config.h

```diff
@@ -17,6 +17,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include "config_common.h"
+#include <stdio.h>
```

Comment:
```suggestion
```

Reply:
I've now moved it to the keymap - if I omit it, it does not compile https://s.0xcb.dev/95oKo